src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+function CheckIfDurableFunctionsEnabled {
+	if (-not [bool]$env:PSWorkerEnableExperimentalDurableFunctions) {
+		throw 'Durable function is not yet supported for PowerShell.'
+	}
+}
+
+<#
+.SYNOPSIS
+    Start an orchestration Azure Function.
+.DESCRIPTION
+    Start an orchestration Azure Function with the given function name and input value.
+    It requires an output binding of the type 'orchestrationClient' is defined for the caller Azure Function.
+    If no such output binding is defined, this function throws an exception.
+.EXAMPLE
+    PS > Start-NewOrchestration -FunctionName starter -InputObject "input value for the orchestration function"
+    Return the instance id of the new orchestration.
+.PARAMETER FunctionName
+    The name of the orchestration Azure Function you want to start.
+.PARAMETER InputObject
+    The input value that will be passed to the orchestration Azure Function.
+#>
+function Start-NewOrchestration {
+    [CmdletBinding()]
+    param(
+        [Parameter(
+            Mandatory=$true,
+            Position=0,
+            ValueFromPipelineByPropertyName=$true)]
+        [string] $FunctionName,
+
+        [Parameter(
+            Position=2,
+            ValueFromPipelineByPropertyName=$true)]
+        [object] $InputObject
```

Comment:
Please add [ValidateNotNull()]

Reply:
No, I think this one should be allowed to be $null.

Reply:
Thanks for clarifying. Resolving conversation.